examples/chat/pages/index.tsx

```diff
@@ -8,30 +8,32 @@ import { defaultVariables } from '@dialectlabs/react-ui/components/common/ThemeP
 // teal: #59C29D
 // dark: #353535
 // light: #F6F6F6
-// border-light: #F0F0F0
+// dt-border-light: #F0F0F0
 // blue: #448EF7
 
 // TODO: Use useTheme instead of explicitly importing defaultVariables
 export const themeVariables: IncomingThemeVariables = {
   dark: {
     bellButton:
-      'w-12 h-12 shadow-xl shadow-neutral-800 border border-neutral-600 hover:shadow-neutral-700',
-    modal: 'rounded-3xl shadow-xl shadow-neutral-900 border border-neutral-800',
+      'dt-w-12 dt-h-12 dt-shadow-xl dt-shadow-neutral-800 dt-border dt-border-neutral-600 hover:dt-shadow-neutral-700',
```

Comment:
Question: does it makes sense to leave the example styles unprefixed, since examples behave as external projects. It would be actually a great test, if we "unprefixed" tailwind styles on top of the prefixed ones (inside react-ui). What do you think?

Reply:
Cool idea, to procced with it we need to build styles for lib and examples separately. Could do in the subsequent PR